src/bandersnatch/delete.py

```diff
@@ -22,20 +22,19 @@
 
 async def delete_path(blob_path: Path, dry_run: bool = False) -> int:
     storage_backend = next(iter(storage_backend_plugins()))
+    loop = asyncio.get_event_loop()
     if dry_run:
         logger.info(f" rm {blob_path}")
         return 0
-    blob_exists = await storage_backend.loop.run_in_executor(
+    blob_exists = await loop.run_in_executor(
```

Comment:
I bet the backend loop is being patch / mocked or something to make the tests work.

So I bet we don't want to do this change and workout why the backend's loop is failing here ...